packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -412,23 +412,28 @@ describe('CLI', () => {
     });
   });
 
-  it('generates mnemonic', () => {
-    const out = execCommand('generate-mnemonic');
+  it('generates mnemonic', async () => {
```

Comment:
Weird - how did this not break before? The generate-mnemonic function was always async 🤔

Reply:
but it's used through CLI using the synchronous `execCommand` so async is not required here.